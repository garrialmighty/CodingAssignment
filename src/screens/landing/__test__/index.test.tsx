import React from 'react';
import {render} from '@testing-library/react-native';

import LandingScreen from '../';

describe('Screen: Landing', () => {
  it('should render properly', () => {
    const {toJSON} = render(<LandingScreen />);
    expect(toJSON).toMatchSnapshot();
  });
});
