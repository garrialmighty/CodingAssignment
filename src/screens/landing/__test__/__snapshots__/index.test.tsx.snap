// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen: Landing should render properly 1`] = `[Function]`;
